packages/token-providers/src/getSsoOidcClient.ts

```diff
@@ -1,23 +1,18 @@
-const ssoOidcClientsHash: Record<string, any> = {};
```

Comment:
I removed this caching functionality. Credentials are cached already, there should be no need to cache the client as well. Caching the client conflicts with being able to configure the client, since this cache exposes no method to clear it.